src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -652,38 +663,38 @@ This endpoint is used for creating a new trip. To create a new trip, the specifi
 
 | Parameter Name |Data Type |Description
 | --------- | --------- | -------
-| userid_type (optional) | string	| The type of user identification to use. Possible value is: `login_id`
-| userid_value (optional) | string	| The value for the user identification type. Currently the only available type is `login_id` so the value is the login credentials. This parameter must be provided in conjunction with the `userid_type` parameter. The `userid_type` and `userid_value` parameters can only be used if the user account associated with the OAuth 2.0 access token is associated with a Concur account with one of these roles: Web Services Administrator for Professional or Can Administer for Standard. The format for the request URI using the `userid_type` and `userid_value` query parameters is `/travel/trip/v1.1/trip_ID?userid_type=login&userid_value=login_ID`
+| userid_type (optional) | string	| The type of user identification to use. Possible value is: `login_id`.
```

Comment:
Let's move "optional" to the description

Reply:
Done